src/app/body/team-details/label-card/label-card.component.ts

```diff
@@ -11,13 +11,15 @@
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
 * See the MIT License for more details. 
 ***********************************************************/
+import { ThrowStmt } from '@angular/compiler';
 import { Component, Input, OnInit, Output } from '@angular/core';
 import { AngularFireFunctions } from '@angular/fire/compat/functions';
 import { map } from 'rxjs';
 import { Label } from 'src/app/Interface/TeamInterface';
 import { BackendService } from 'src/app/services/backend/backend.service';
 import { ErrorHandlerService } from 'src/app/services/error-handler/error-handler.service';
 import { EventEmitter } from 'stream';
+import { threadId } from 'worker_threads';
```

Comment:
This is also not needed

Reply:
removed both lines 21 and 22 as they were unused imports